app/assets/stylesheets/alchemy/elements.scss

```diff
@@ -320,8 +320,10 @@ div#cells {
   }
 }
 
-a.expand_element {
-  background: image-url('alchemy/icons.png') no-repeat -160px -72px;
+.expand_element,
```

Comment:
Selector `expand_element` should be written in lowercase with hyphens<br>Selector `fold_element` should be written in lowercase with hyphens